perfkitbenchmarker/packages/fio.py

```diff
@@ -100,6 +102,28 @@ def ParseJobFile(job_file):
   return parameter_metadata
 
 
+def FioParametersToJob(fio_parameters):
+  """Translate fio parameters into a job file in raw string.
+
+  Sample fio parameters:
+  --filesize=10g --directory=/scratch0
+  --ioengine=libaio --filename=fio_test_file --invalidate=1
+  --randrepeat=0 --direct=0 --size=3790088k --iodepth=8
+  --name=sequential_write --overwrite=0 --rw=write --end_fsync=1
+
+  Args:
+    fio_parameter: string. Fio parameters in string format.
+
+  Returns:
+    A raw string representing a job file can be read by fio binary.
+  """
+  fio_parameters = fio_parameters.replace(' ', '\n')
+  fio_parameters = regex_util.Substitute(
+      CMD_SECTION_REGEX, JOB_SECTION_REGEX, fio_parameters)
+  fio_parameters = '[%s]\n%s' % (GLOBAL, fio_parameters)
+  return fio_parameters.replace('--', '')
```

Comment:
Recommend a regex here in case someone has `--` in one of their parameter values.